_posts/2023-02-06-docs-datascienceprojects.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: blog
+author: ezidav
+title:  New documentation for Data Science Projects added
+preview: Added New Data Science Project documentation
+date: 2023-02-06
+categories: features, release, documentation
+---
+
+New documentation has been added for the new Data Science Projects feature in the ODH dashboard,the documentation covers the following items:
```

Comment:
Can you add a link to the DSP documentation so that viewers can go directly to the documentation?

http://opendatahub.io/docs/working-on-data-science-projects.html